rosbag2_py/CMakeLists.txt

```diff
@@ -96,6 +103,7 @@ ament_target_dependencies(_reindexer PUBLIC
 # Install cython modules as sub-modules of the project
 install(
   TARGETS
+    _compression_options
```

Comment:
I wanted to call this `_compression` but there is a collision with some other internal python library, which leads to this error on compilation.

```
--- stderr: rosbag2_py                           
Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/usr/lib/python3.8/compileall.py", line 332, in <module>
    exit_status = int(not main())
  File "/usr/lib/python3.8/compileall.py", line 230, in main
    parser = argparse.ArgumentParser(
  File "/usr/lib/python3.8/argparse.py", line 1696, in __init__
    self.add_argument(
  File "/usr/lib/python3.8/argparse.py", line 1394, in add_argument
    self._get_formatter()._format_args(action, None)
  File "/usr/lib/python3.8/argparse.py", line 2493, in _get_formatter
    return self.formatter_class(prog=self.prog)
  File "/usr/lib/python3.8/argparse.py", line 178, in __init__
    import shutil as _shutil
  File "/usr/lib/python3.8/shutil.py", line 22, in <module>
    import bz2
  File "/usr/lib/python3.8/bz2.py", line 30, in <module>
    class BZ2File(_compression.BaseStream):
AttributeError: module '_compression' has no attribute 'BaseStream'
```

Reply:
@asymingt Is it still a problem?  
I don't see this failure on the rpr job https://build.ros2.org/job/Rpr__rosbag2__ubuntu_jammy_amd64/935/